### ISSUE TITLE
conctact cylinder error: There is no sub model part with name "ComputingContact" in model part "Structure"

### ISSUE THREAD

**Issue by luchete80** [KratosMultiphysics/Kratos#12813]
**Description**
After some corrections to ContactStructuralMechanics app,  I've managed to run an old example: contactcylinders with remeshing. 
This is documented in repo https://github.com/luchete80/Kratos_tutorials/
I get: 

RuntimeError: Error: There is no sub model part with name "ComputingContact" in model part "Structure"
The following sub model parts are available:
	"Contact"
	"GENERIC_FixedBase"
	"GENERIC_Displacement_Auto1"
	"SkinModelPart"
	"CONTACT_Contact_master_Auto1"
	"CONTACT_Contact_slave_Auto1"
	"Parts_Solid_Auto1"
	"Parts_Solid_Auto2"

This is happening on search_base_process.py, line 223 which clear and regenerate the subModelPart "Contact". 

If I replace subpart "Contact" by  "ComputingContact" i get:

    self.search_utility_list[key] = CSMA.ContactSearchProcess(self.main_model_part, search_parameters, properties_pair)
RuntimeError: Error: AdvancedContactSearch:: Please add the Contact submodelpart to your modelpart list

On the other hand, I see that test_dynamic_search.py

**Scope**
Where in Kratos is this bug happening. 
E.g.
- ContactStructuralMechanicsApplication

**To Reproduce**
Run the (added here) files:
https://github.com/luchete80/Kratos_tutorials/tree/master/contacting_cylinders_fixin
I've replaced the following fropm branch 9.5:
https://github.com/luchete80/Kratos_tutorials/blob/master/Fixings_Remeshing

**Expected behavior**
Remesh and continue

**Environment**
- OS: Ubuntu 22.04
- Branch: Kratos-9.5 release
- Python 3.11

**Comment by loumalouomega** [comment]
Sorry contact with remeshing is currentñy broken and we could not agree in an interface. This was done in a branch that does not compile anymore (2019 branch)

**Comment by luchete80** [comment]
Uhm ok, I will try to run it on that one.
Thank you very much!!!

**Comment by loumalouomega** [comment]
> Uhm ok, I will try to run it on that one. Thank you very much!!!

The branch was private and lost

**Comment by luchete80** [comment]
Oh, what a pity. Is a chance to access to a branch from this current repo with similar characteristics? Thanks

**Comment by loumalouomega** [comment]
Unfortunately not, I uploaded an intermediate branch but it is broken, and the one actually working was in a private computer that I not longer have with me (it was 5 years+ ago)

**Comment by loumalouomega** [comment]
I could try to revive the thng, but I would like to agree an interface before any job @rubenzorrilla FYI

**Comment by rubenzorrilla** [comment]
Agree. We can discuss and agree about an upgraded interface and implementation. However, unfortunately at the moment we have no resources to allocate here.

**Comment by loumalouomega** [comment]
> Agree. We can discuss and agree about an upgraded interface and implementation. However, unfortunately at the moment we have no resources to allocate here.

I will think a new design and propose it.
